src/utils/getDefaultCommitContent.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import fs from 'fs'
+
+const getDefaultCommitContent = () => {
+  try {
+    const commitFilePath: string = process.argv[3]
```

Comment:
I think we need to clarify or move those magic numbers into constants that explain why we use those values